lib/_http_client.js

```diff
@@ -707,21 +707,15 @@ ClientRequest.prototype.setTimeout = function setTimeout(msecs, callback) {
   return this;
 };
 
-ClientRequest.prototype.setNoDelay = function setNoDelay() {
-  const argsLen = arguments.length;
-  const args = new Array(argsLen);
-  for (var i = 0; i < argsLen; i++)
-    args[i] = arguments[i];
-  this._deferToConnect('setNoDelay', args);
-};
-ClientRequest.prototype.setSocketKeepAlive = function setSocketKeepAlive() {
-  const argsLen = arguments.length;
-  const args = new Array(argsLen);
-  for (var i = 0; i < argsLen; i++)
-    args[i] = arguments[i];
-  this._deferToConnect('setKeepAlive', args);
+ClientRequest.prototype.setNoDelay = function setNoDelay(noDelay) {
+  this._deferToConnect('setNoDelay', [noDelay]);
 };
 
+ClientRequest.prototype.setSocketKeepAlive = _setSocketKeepAlive;
```

Comment:
I think the underscore should be removed here, so that the function name matches the prototype property name.